plugins/package-managers/node/src/main/kotlin/yarn2/Yarn2.kt

```diff
@@ -185,7 +184,7 @@ class Yarn2(override val descriptor: PluginDescriptor = Yarn2Factory.descriptor,
         }.toMap()
     }
 
-    private fun getRemotePackageDetails(workingDir: File, moduleIds: Set<String>): Set<PackageJson> =
+    private fun getRemotePackageDetails(moduleIds: Set<String>): Set<PackageJson> =
```

Comment:
I know this is a simplification that I've done myself elsewhere, and I'm fine with it. However, it later occurred to me that the working directory *might* be important in cases where a project-local `.yarnrc` file (or similar) configures credentials for private registries. We should keep tat in mind in case we run into failures caused by that.

Reply:
This indeed made the tests fail, so I dropped it. The underlying error is `Usage Error: No project found in ...`.